.github/workflows/test.yml

```
@@ -0,0 +1,46 @@
+name: Test Wadm (v0.4)
+
+on:
+  pull_request:
+    branches: [ wadm_0.4 ]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04 ]
```

```suggestion
        os: ubuntu-22.04
```

1. There's only one OS specified, so no need for the list
2. `22.04` is the latest, unless there is a reason to use `20.04`, which would need to be documented here

---

Hey thanks for the feedback -- I don't think that there *should* be only one OS for now (the other `wadm` workflows have more), and the other wadm tests use `20.04` rather than `22.04` which is why I did the same thing here. 

Super open to changing this though -- @thomastaylor312  @autodidaddict can I update this bit?

---

OK going to just change this!